### DISCUSSION TITLE
The Monorepo Handbook: how to handle peerDependencies between packages?

### DISCUSSION THREAD

**Discussion by giancarlosisasi** [vercel/turborepo#2254] [Help] [answered]
Hi there
I was using turbo repo trying to implement a monorepository that contains apps and a UI framework. But then I found this problem with pnpm:

- https://github.com/pnpm/pnpm/issues/5479
- https://github.com/pnpm/pnpm/discussions/3938

I know it's not related 100% to turbo but I wanted to create this because maybe we can add it to the monorepo handbook.
So other people are aware of this problem.

**Comment by nathanhammond** [comment]
@mattpocock the request here is to inline some portion of these docs: https://pnpm.io/package_json#dependenciesmetainjected

**Reply by mattpocock** [reply, marked as answer]
@GiancarlosIO I think the handbook should probably focus on an overview, instead of specific issues for specific tools. I see it as an entry point, not necessarily an encyclopedia. If this discussion gets more upvotes, indicating that this is a big issue, then that'll likely change my mind.

**Comment by giancarlosisasi** [comment]
Related issue: https://github.com/vercel/turbo/issues/2306
